doc/plutip-testing.md

```diff
@@ -44,26 +50,128 @@ CTL can help you test the offchain `Contract`s from your project (and consequent
 
 There are two approaches to writing such tests.
 
-[First](#testing-in-aff-context) is to use the `Contract.Test.Plutip.runPlutipContract` function, which takes a single `Contract`, launches a Plutip cluster and executes the passed contract.
-This function runs in `Aff`; it will also throw an exception should contract fail for any reason.
-After the contract execution the Plutip cluster is terminated.
-You can either call it directly from your test's main or use any library for grouping and describing tests which support effects in the test body, like Mote.
+### Testing with Mote overview
 
 [Mote](https://github.com/garyb/purescript-mote) is a DSL for defining and grouping tests (plus other quality of life features, e.g. skipping marked tests).
 
-
-[Second](#testing-with-mote) (and more widely used) approach is to first build a tree of tests (in CTL's case a tree of `ContractTest` types -- basically a function from some distribution of funds to a `Contract a`) via Mote and then use the `Contract.Test.Plutip.testPlutipContracts` function to execute them.
+First (and more widely used) approach is to first build a tree of tests (in CTL's case a tree of `ContractTest` types -- basically a function from some distribution of funds to a `Contract a`) via Mote and then use the `Contract.Test.Plutip.testPlutipContracts` function to execute them.
 This allows to set up a Plutip cluster only once per top-level groups and tests passed to the `testPlutipContracts` and then use it in many independent tests.
 The function will interpret a `MoteT` (effectful test tree) into `Aff`, which you can then actually run.
 
 The [`ctl-scaffold` template](../templates/ctl-scaffold) provides a simple `Mote`-based example.
 
+More info and usage examples are [here](#testing-with-mote).
+
+### Testing in Aff context overview
+
+[First](#testing-in-aff-context) is to use the `Contract.Test.Plutip.runPlutipContract` function, which takes a single `Contract`, launches a Plutip cluster and executes the passed contract.
```

Comment:
This is duplicate. "First" is used above already in a similar paragraph.

Reply:
Fixed